### PR TITLE
5 minute ssh timeout

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #5572 

## Summary

**[Timeouts on Pantheon](https://pantheon.io/docs/timeouts#timeouts-that-are-not-configurable)** - Clarifies that SSH times out on the Platform after 5 minutes of inactivity.

## Remaining Work

The following changes still need to be completed:

- [ ] ~confirm `60 minute hard limit`~
- [x] reviewed date?

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5577 | Difference |
| --- | --- | --- | --- |
| Performance Score | `44` | `43` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5577` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19031/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19031/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/timeouts.md [hunk 1]
```diff
@@ -24,7 +23,7 @@ Rules are for the good of the group, and timeouts are no exception. Timeouts are
...
 | [PHP set_time_limit](https://secure.php.net/manual/en/function.set-time-limit.php)                                           | 120 seconds | Number of seconds a script can run. If reached, the script returns a fatal error. |
 | Load Balancer                                                                                                                | 120 seconds | Applies to HTTPS requests and requests to a DNS A record. Requests using the Pantheon CNAME for HTTP requests are *not* limited |
-| SSH                                                                                                                          | 10 minutes with no communication <br /> 60 minutes hard limit | Applies to remote Drush and WP-CLI commands, SSH tunneling, SFTP, rsync |
+| SSH                                                                                                                          | 5 minutes with no communication; <br /> 60 minute hard limit | Applies to remote Drush and WP-CLI commands, SSH tunneling, SFTP, rsync  |
```
Comment (spheromak) [inline source/content/timeouts.md]: this is not what we changed. we changed an authentication timeout.  I think that requires new copy / entry here 

TImeouts:
Authentication - 5 mins  
idle  - 10 mins  
Currently there is _hard_ timeout on active sessions but I am ok with folks being told there is 1 hour.

Comment (spheromak) [reply source/content/timeouts.md]: there are other limitations

per user there are only allowed max 20 connections to the same site-env. Tho this is more of a protection against denial of servic. We haven't seen a user trigger this limit in our metrics from the last 3 months.

Similarly there is a total 500 active sessions per node in the platform. This too is to protect us from malicious or accidental DOS. Again we haven't actually see this limit hit in practice.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5577 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `56.00000000000001` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5577` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19075/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19075/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5577 | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `48` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5577` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19079/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19079/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/timeouts.md (hunk 1 above)
Comment (EdwardAngert) [reply source/content/timeouts.md]: thanks, @spheromak!
I put that `10` back, and adjusted for the no-hard timeout to reflect current Platform behavior.

for the other points about max connections and active sessions, I think we can consider other docs to include those in if it's something our users or CSEs might encounter, and I'm going to leave it out of this PR for now as it's outside of the `timeouts` need.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5577 | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `44` | `-6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5577` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19106/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19106/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**